tests/0081-admin.c

```diff
@@ -2796,14 +2996,22 @@ static void do_test_ListConsumerGroups(const char *what,
                 is_simple_consumer_group =
                     rd_kafka_ConsumerGroupListing_is_simple_consumer_group(
                         group);
-                state = rd_kafka_ConsumerGroupListing_state(group);
+                state      = rd_kafka_ConsumerGroupListing_state(group);
+                group_type = rd_kafka_ConsumerGroupListing_type(group);
                 for (j = 0; j < TEST_LIST_CONSUMER_GROUPS_CNT; j++) {
                         if (!strcmp(list_consumer_groups[j], group_id)) {
                                 found++;
                                 TEST_ASSERT(!is_simple_consumer_group,
                                             "expected a normal group,"
                                             " got a simple group");
 
+                                if (has_match_types)
+                                        TEST_ASSERT(
+                                            group_type ==
```

Comment:
Yes, these are true but since the test is similar for other clients which do not have has_match_types functionality like in librdkafka, so only these were agreed upon. Otherwise, we would diverge a lot !